test/Makefile.am

```diff
@@ -0,0 +1,3 @@
+test:
+	$(MAKE) -C ../src/retrace config.py
+	PATH=${abs_top_srcdir}/src:$(PATH) RETRACE_SERVER_PLUGIN_DIR=${abs_top_srcdir}/src/plugins RETRACE_SERVER_CONFIG_PATH=${abs_top_srcdir}/src/config/retrace-server.conf python run_test.py --coredump=$(coredump) --executable=$(executable) --package=$(package) --release=$(release) $(ARGS)
```

Comment:
Yes, you can run the test without any arguments. But these argument are here for one case - when you want to specify your own coredump (you don't want coredump to be generated, but use the one, you have) and therefore you have to specify package and executable.